core/common/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/share/ShareManager.kt

```diff
@@ -0,0 +1,14 @@
+package io.github.droidkaigi.confsched2023.share
+
+/**
+ * Platform sharing feature wrapper
+ */
+interface ShareManager {
```

Comment:
Maybe we won't use this this year for other platforms. So how about removing this and rename AndroidShareManager with `ShareStarter`or `ShareLauncher` or `ShareNavigator`?